python/cog/command/openapi_schema.py

```diff
@@ -3,6 +3,7 @@
 
 This prints a JSON object describing the inputs of the model.
 """
+
```

Comment:
```suggestion
```

Reply:
this change required by: python -m ruff format --check python
not changing.